### ISSUE TITLE
Remove Uint64List support from polkadart_scale_codec and polkadart_cli packages

### ISSUE THREAD

**Issue by Lohann** [leonardocustodio/polkadart#297]
## Context
Dart Web doesn't support `Uint64List`, and some packages rely in this data structure, we must remove Uint64List references otherwise `polkadart_scale_codec` cannot be used on the web. Change all ocurrences to lists of BigInt.

- https://github.com/dart-lang/sdk/issues/50332

## Acceptance Criteria
- Remove `Uint64List` and `Int64List` support from `u64`  and `i64` codecs.
- Refactor `polkadart_cli` for not use `U64Sequence`, `I64Sequence`, `U64Array` and `I64Array` codecs.
- Unit tests passes
- Update CHANGELOG.md  for both packages.

**Comment by avive** [comment]
Thank you for this lovely lib! We consider using it for Karma Coin mobile app. Can you guys please confirm that supporting web apps is on the project short-term intention to support web apps and fix this issue?

**Comment by Lohann** [comment]
@avive yes! the plan is fully support all platforms supported by flutter, currently the only blocker for web is this issue here, which will be fixed shortly.

**Comment by leonardocustodio** [comment]
Solved by #508
